### PR TITLE
Create conan recipe, Impl GitHub CI workflow for package create/upload

### PR DESCRIPTION
**Pull request by dBagrat**

### CODE REVIEW

**Review by sssilver — APPROVED** [review]

File: .github/workflows/conan.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        submodules: 'recursive'
+    - name: Install dependencies
+      run: sudo pip install conan
+    - name: Config
+      env:
+        CONAN_LOGIN_USERNAME: ${{ secrets.JFROG_USER }}
+        CONAN_PASSWORD: ${{ secrets.JFROG_APIKEY }}
+      run: |
+        conan remote add triada https://triada.jfrog.io/artifactory/api/conan/conan
+        conan user -r triada -p --skip-auth
```
Comment (sssilver) [inline .github/workflows/conan.yml:24]: how does auth work here?

Comment (dBagrat) [reply .github/workflows/conan.yml:24]: It causes conan to skip interactive authentication process and uses `CONAN_LOGIN_USERNAME` and `CONAN_PASSWORD` for authentication later

File: conanfile.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools
+import os
+
+class FastNoiseLiteConan(ConanFile):
+    name = "fastnoise-lite"
+    version = "2021.11.29"
```
Comment (andranik3949) [inline conanfile.py]: Just writing this comment so we don't forget to change the versioning

**Review by andranik3949 — APPROVED** [review]

File: conanfile.py (hunk 2 above)
Comment (dBagrat) [reply conanfile.py]: 1.0.3.2111 ?

**Pull request merged**